modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-bootstrap-vm-inspecting-logs_{context}"]
+= Inspecting logs
+
+When experiencing issues downloading or accessing the RHCOS images, first verify that the URL is correct in the `install-config.yaml` configuration file.
+
+.Example of internal webserver hosting RHCOS images
+[source,yaml]
+----
+bootstrapOSImage: http://<ip:port>/rhcos-43.81.202001142154.0-qemu.x86_64.qcow2.gz?sha256=9d999f55ff1d44f7ed7c106508e5deecd04dc3c06095d34d36bf1cd127837e0c
+clusterOSImage: http://<ip:port>/rhcos-43.81.202001142154.0-openstack.x86_64.qcow2.gz?sha256=a1bda656fa0892f7b936fdc6b6a6086bddaed5dafacedcd7a1e811abb78fe3b0
+----
+
+The `ipa-downloader` and `coreos-downloader` containers download resources from a webserver or the external `https://quay.io` registry, whichever is specified in the `install-config.yaml` configuration file. Verify the following two containers are up and running and inspect their logs as needed:
```

Comment:
If you don't want quay.io to be a hyperlink, use this format:
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#example-urls